Various fixes (#4193)
umbraco/UmbracoDocs: Reference/Configuration-for-Umbraco-7-and-8/umbracoSettings/index-v7.md

<EnablePropertyValueConverters>true</EnablePropertyValueConverters>
```

## `<loginBackgroundImage>` (Available on Umbraco versions 7.6.0 and above)

You can specify your own background image for the login screen here. The image will automatically get an overlay to match backoffice colors. This path is relative to the ~/umbraco path. The default location is: /umbraco/assets/img/installer.jpg

```xml
<loginBackgroundImage>../App_Plugins/Backgrounds/login.png</loginBackgroundImage>
```

## `<AllowedUploadFiles>` (Available on Umbraco versions 7.6.2 and above)

> If greater control is required than available from the above, this setting can be used to store a "white list" of file extensions.  If provided, only files with these extensions can be uploaded via the backoffice.

Reviewer: ```suggestion
If greater control is required than available from the above, this setting can be used to store an "allow list" of file extensions.  If provided, only files with these extensions can be uploaded via the backoffice.
```
